packages/language-server/test/copmletion-items-classes-spec.ts

```diff
@@ -21,6 +25,62 @@ describe("the UI5 language assistant Code Completion Services - classes", () =>
     ui5SemanticModel = await generateModel({ version: "1.74.0" });
   });
 
+  /** The first (but not final) place the custor stops when inserting the completion. Pressing tab moves it to the next place. */
+  const TAB_STOP1 = "${1}";
+
+  interface ClassCompletionItem {
```

Comment:
was not this code in another PR? does this PR need to be updated? I will try to do so